backends/cadence/aot/functions_hifi.yaml

```diff
@@ -25,7 +25,7 @@
 - op: add.out
   kernels:
     - arg_meta: null
-      kernel_name: torch::executor::add_out
+      kernel_name: impl::HiFi::add_out
```

Comment:
probably should be cadence::impl::HiFi too, but we can do that in a follow up if this can be merged

Reply:
I will create a new PR for this

Reply:
https://github.com/pytorch/executorch/pull/6433